pkg/proxy/util/utils.go

```diff
@@ -477,6 +477,14 @@ func WriteLine(buf *bytes.Buffer, words ...string) {
 	}
 }
 
+// WriteRuleLine joins all words with spaces, terminate with newline and write to buff.
+func WriteRuleLine(buf *bytes.Buffer, chainName string, words ...string) {
```

Comment:
You just copied the comment from `WriteLine` but forgot to update it. Need to add something about "prepends "-A" and chainName to words" or something like that.

Reply:
done.